### PR TITLE
New skips added from Beginning-Djose, improving and fixing some skips

### PR DESCRIPTION
**Pull request by flobberworm4**

### CODE REVIEW

File: FFXCutsceneRemover/Components/MemoryWatchers.cs [hunk 1]
```diff
@@ -45,6 +45,16 @@ sealed class MemoryWatchers
...
 
+        public MemoryWatcher<byte> EnableAuron;
+
+        public MemoryWatcher<byte> LucaFlag;
+        public MemoryWatcher<byte> LucaFlag2;
+
+        public MemoryWatcher<byte> MiihenFlag1;
+        public MemoryWatcher<byte> MiihenFlag2;
+        public MemoryWatcher<byte> MiihenFlag3;
+        public MemoryWatcher<byte> MiihenFlag4;
```
Comment (flobberworm4) [inline FFXCutsceneRemover/Components/MemoryWatchers.cs:56]: Not sure how feasible this is but a MemoryWatcher<byte[]> array will be useful for these flags that are close to each other in memory. Also needed for sorting out formations

Comment (erickt420) [reply FFXCutsceneRemover/Components/MemoryWatchers.cs:56]: Let's chat about what this implementation might look like offline.

File: FFXCutsceneRemover/Components/MemoryWatchers.cs [hunk 2]
```diff
@@ -119,7 +138,14 @@ public void Initialize(Process process)
...
                     HpEnemyA,
-                    GuadoCount
+                    GuadoCount,
+                    EnableAuron,
+                    LucaFlag,
+                    LucaFlag2,
+                    MiihenFlag1,
+                    MiihenFlag2,
+                    MiihenFlag3,
+                    MiihenFlag4
```
Comment (erickt420) [inline FFXCutsceneRemover/Components/MemoryWatchers.cs:148]: Is it possible to provide more useful names for these guys?

Like for example one of the mi'ihen flags skips the convo with Shelinda, right? Can we call it ShelidaMiihenFlag or something like this?

**Review by erickt420 — COMMENTED** [review]
Looks real nice overall. If you agree with my variable naming feedback, I am okay with fixing these in a future commit.

**Pull request merged**